client/src/templates/Introduction/components/block.tsx

```diff
@@ -122,7 +122,8 @@ export class Block extends Component<BlockProps> {
     const isProjectBlock = challenges.some(({ challenge }) => {
       const isJsProject =
         [3, 6, 10, 14, 17].includes(challenge.order) &&
-        challenge.challengeType === 5;
+        challenge.challengeType === 5 &&
+        challenge.superBlock !== 'coding-interview-prep';
```

Comment:
🙄

Reply:
We need to rework our challengeTypes - each type should have it's own type. The JS projects are the same type as the rest of the JS challenges, it should be its own.